CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- EXAMPLE
+
+## [2.0.0-alpha]
+
+### Added
+
+- I've added feature XY (#1000)
+
+### Changed
+
+- I've cleaned up XY (#1000)
+
+### Deprecated
+
+- I've deprecated XY (#1000)
+
+### Removed
+
+- I've removed XY (#1000)
+
+### Fixed
+
+- I've fixed XY (#1000)
+
+### Security
+
+- I've improved the security in XY (#1000)
+
+-->
+
+## [Unreleased]
+
+## [2.0.0-alpha.0]
+
+[Unreleased]: https://github.com/ethereum/web3.js/compare/v1.0.0-beta.55...HEAD
```

Comment:
This link is going to be updated with each release?

Reply:
Yes I thought it could be updated on each release PR but I will remove it because the release PR template does already have a compare view.